lambdas/service/app.py

```diff
@@ -1662,21 +1764,7 @@ def generate_manifest(event: AnyJSON, _context: LambdaContext):
                 made. If that fails, the UUID of the file will be used instead.
             ''')
         ),
-        params.query(
-            'wait',
-            schema.optional(int),
```

Comment:
> Why was this changed from int to string?

This was a mistake. I was confused by the string form of the param in the request. I've set it back to `int` in the schema.

> Why was the `default` logic added? I don't think we have a use case for there not being a default.

We do not set a `wait` default value for the `/repository/files/{file_uuid}` and `/fetch/repository/files/{file_uuid}` endpoints.
Also, a default value is added for the initial manifest request (POST `/manifest/files`), but not for the subsequent requests to GET `/manifest/files/{token}` 

> REVIEW: Why was the link removed?

Links are not supported in this section of the Swagger
From: https://service.azul.data.humancellatlas.org/#/Manifests/put_manifest_files

![Screenshot 2024-08-16 at 5 52 09 PM](https://github.com/user-attachments/assets/c40a1bda-2f40-4667-b8e0-a6149d8386ba)

Reply:
A "default" is a value that takes effect when no explicit value is specified. When an endpoint accepts a `wait` parameter, that parameter must be optional so as to not complicate "regular" use cases in which the wait can be performed client-side. For every optional parameter a default needs to be specified. Anything else would lead to incorrect or incomplete documentation. If you are unable to implement this directive for any reason, or if you think I am wrong, please request a PL slot so we can clear this up.

Reply:
Added a default value for the `wait` parameter in both the `/repository/files` and `/manifest/files` endpoints.